src/main/java/com/easyrest/facade/admin/ActionOnUserFacade.java

```diff
@@ -0,0 +1,46 @@
+package com.easyrest.facade.admin;
+
+import com.easyrest.components.admin.ActionOnUser;
+import com.easyrest.components.admin.LeftSideMenu;
+import org.openqa.selenium.WebDriver;
+
+public class ActionOnUserFacade {
+    private WebDriver driver;
```

Comment:
Do you need driver as an instance variable?

Reply:
> Do you need the driver as an instance variable?

yes

Reply:
Marta, why do you need a driver on the facade?

Reply:
> Do you need the driver as an instance variable?

I reviewed, don t need driver, fix it